HousingSearchApi.Tests/V1/Helper/ProcessListSortFactoryTests.cs

```diff
@@ -0,0 +1,108 @@
+using FluentAssertions;
+using Hackney.Shared.HousingSearch.Gateways.Models.Processes;
+using HousingSearchApi.V1.Boundary.Requests;
+using HousingSearchApi.V1.Infrastructure.Sorting;
+using Xunit;
+
+namespace HousingSearchApi.Tests.V1.Helper
+{
+    public class ProcessListSortFactoryTests
+    {
+        private SortFactory _sut;
+
+        public ProcessListSortFactoryTests()
+        {
+            _sut = new SortFactory();
+        }
+
+        [Fact]
+        public void ShouldNotSortAsDefault()
+        {
+            // Arrange + Act
+            var result = _sut.Create<QueryableProcess, GetProcessListRequest>(new GetProcessListRequest());
```

Comment:
Can you add a test or another case for if SortBy is an empty string?

Reply:
it will hit the DefaultSort as there will be no matching with the sortBy values and I think this test is covered by the first case from that file